s3_helper.py

```diff
@@ -0,0 +1,176 @@
+"""
```

Comment:
Since we're starting to reuse some of these files, maybe we can consider a library repository and import it with pip.

Reply:
Will work on this when we make the sandbox exporter methods importable.

Reply:
Sounds good